src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminal.accessibility.contribution.ts

```diff
@@ -111,6 +124,21 @@ export class TerminalAccessibleViewContribution extends Disposable implements IT
 		}));
 	}
 
+	private _updateCommandExecutedListener(): void {
+		if (!this._instance.capabilities.has(TerminalCapability.CommandDetection)) {
+			return;
+		}
+		if (!this._configurationService.getValue(TerminalSettingId.AccessibleViewFocusOnCommandExecution)) {
+			this._onCommandExecutedShowListener.clear();
+			return;
+		} else if (this._onCommandExecutedShowListener.value) {
+			return;
+		}
+
+		const capability = this._instance.capabilities.get(TerminalCapability.CommandDetection)!;
+		this._onCommandExecutedShowListener.value = this._register(capability.onCommandExecuted(() => this.show()));
```

Comment:
I don't think you want to show if if the terminal was not previously focused, for example run a terminal command from some command (arguable what's the right behavior) or multiple commands are run in sequences like pasting `long command \nanother command` (definitely undesirable).